.github/workflows/Build_and_publish_package.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and Deploy Packages
+
+on: [push]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: RelWithDebInfo
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#configuring-a-build-matrix
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Update git submodule
+      # ensure that the libgit2 linkage is correct
+      run: |
+        git submodule init
+        git submodule update
```

Comment:
Instead of this step, you can add parameters to checkout@v2 action:

```
      with:
        submodules: recursive
```

Reply:
removed Update git submodule step. added with submodules recursive parameter to the build script